MovieApp/MovieCategoriesListViewController.swift

```diff
@@ -73,6 +79,10 @@ extension PopularMoviesViewController: UITableViewDataSource {
         let categoryId = indexPath.row
         cell.set(title: titleCategories[categoryId], movies: movieCategories[categoryId])
         
+        cell.tapOnMovieCell = {
```

Comment:
Can you not use a method on the collection view, `didSelectItemAt` and remove the need for that handler?